benchmark/src/test/java/com/datatorrent/benchmark/memsql/MemsqlInputBenchmarkTest.java

```diff
@@ -77,8 +77,8 @@ public void testMethod() throws SQLException, IOException
     attributeMap.put(OperatorContext.PROCESSING_MODE, ProcessingMode.AT_LEAST_ONCE);
     attributeMap.put(OperatorContext.ACTIVATION_WINDOW_ID, -1L);
     attributeMap.put(DAG.APPLICATION_ID, APP_ID);
-    OperatorContextTestHelper.TestIdOperatorContext context =
-        new OperatorContextTestHelper.TestIdOperatorContext(OPERATOR_ID, attributeMap);
+    OperatorContextTestHelper.MockOperatorContext context =
```

Comment:
You mean changing individual tests to use the OperatorContext interface instead of MockOperatorContext. I'm OK with it but I don't want this PR to be blocked. If you can complete it today then fine, otherwise please work on it separately.